src/popup/components/Modals/PasswordLogin.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <Modal
+    class="password-login"
+    from-bottom
+    centered
+  >
+    <div class="content-wrapper">
+      <div class="icon-wrapper">
+        <IconBoxed
+          :icon="LockIcon"
+          bg-colored
+          bg-more-transparent
+          icon-padded
+        />
+      </div>
+      <div class="info">
+        <h3 class="text-heading-4 heading">
+          {{ $t('pages.secureLogin.enterPassword') }}
+        </h3>
+
+        <div class="text-description">
+          {{ $t('pages.secureLogin.toUnlock') }}
+        </div>
+      </div>
+      <InputPassword
+        v-model="password"
+        data-cy="password"
+        autofocus
+        class="password-input"
+        :placeholder="$t('pages.secureLogin.login.placeholder')"
+        :label="$t('pages.secureLogin.login.label')"
+        :message="isAuthFailed ? $t('pages.secureLogin.login.error') : null"
+        @keydown.enter="login"
+        @input="isAuthFailed = false"
+      />
+
+      <LinkButton
+        class="forgot-password"
+        href="#"
```

Comment:
I think might break something on mobile. See the `LinkButton`'s `onClick` logic which tries to open the `href` value in new window.